src/app/events/event-group/event-group.component.ts

```diff
@@ -0,0 +1,121 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { Subscription } from 'rxjs';
+import { TranslateService } from '@ngx-translate/core';
+import { NotificationService } from '../../_services/notification.service';
+import { EventDto } from '../../_models/eventDto';
+import { EventGroupService } from '../../_services/api/event-group.service';
+import { EventService } from '../../_services/api/event.service';
+import { EventGroupDto } from '../../_models/eventGroupDto';
+import { EVENT_GROUPS_FORM_ID } from '../../app.constants';
+import { FORM_DATA_EVENT_GROUP } from './event-group-form-data';
+import { FormBase } from '../../shared/dynamic-form/types/form-element-base';
+import { FormElementControlService } from '../../_services/form-element-control.service';
+
+@Component({
+  selector: 'app-event-group',
+  templateUrl: './event-group.component.html',
+  styleUrls: ['./event-group.component.scss'],
+})
+export class EventGroupComponent implements OnInit, OnDestroy {
+  eventGroup: EventGroupDto;
+  events: EventDto[];
+  eventGroupId: any;
+  myFormElements: FormBase<any>[] = [];
+  formData = FORM_DATA_EVENT_GROUP;
+  formId = EVENT_GROUPS_FORM_ID;
+  loading = false;
+
+  subscriptions: Subscription[] = [];
+
+  constructor(
+    public eventGroupService: EventGroupService,
+    private eventService: EventService,
+    private activeRoute: ActivatedRoute,
+    private notificationService: NotificationService,
+    private router: Router,
+    private formElementControlService: FormElementControlService,
+    private translateService: TranslateService
+  ) {}
+
+  ngOnInit(): void {
+    const routeParams = this.activeRoute.snapshot.params;
+    this.eventGroupId = routeParams.eventGroupId;
+    this.fetchGroup();
```

Comment:
shouldn't we use forkJoin here? so the group and events be loaded in the same time